lib/Dialect/Torch/Transforms/DecomposeComplexOps.cpp

```diff
@@ -1184,23 +1184,23 @@ class DecomposeAtenSoftplusOp : public OpRewritePattern<AtenSoftplusOp> {
     Value input = op.self();
     BaseTensorType inputType = input.getType().cast<BaseTensorType>();
 
-    Value inputTimesBeta = rewriter.create<AtenMulScalarOp>(
```

Comment:
the changes in this file are not needed, right?

Reply:
I think when I ran clang format it got autoformatted. 
The indentation might have been screwed in these files. 
I think that's the only change. That should be okay ?

Reply:
It's possible that `git clang-format` considered more commits than just the ones you had in your branch. The normal behavior for `git clang-format` is to only modify your changes. It is a bit of a minor thing, but I would prefer to remove these changes from this PR, so that if anyone needs to do `git blame` on this file to find the commit that added the code, they don't get an extra level of indirection.

Reply:
Yeah, from looking at your branch, it seems that you ran `clang-format` on your commit and the two commits before it,

Reply:
Ah cool makes sense.

Reply:
Fixed.